src/nexus2-client.ts

```diff
@@ -26,20 +26,23 @@ export class Nexus2Client {
   private instance: AxiosInstance
 
   constructor(private nexusServer: NexusServer) {
-    this.instance = axios.create({
-      auth: {
-        username: nexusServer.username,
-        password: nexusServer.password
-      },
+    const config = {
       timeout: nexusServer.timeout,
       maxBodyLength: Infinity,
       maxContentLength: Infinity,
       headers: {
         'Content-Type': 'application/json',
-        Accept: 'application/json'
+        Accept: 'application/json',
+        Authorization: `Bearer ${nexusServer.token}`
```

Comment:
I think there's a check missing. What happens if `nexusServer.token` isn't set? Wouldn't that add a `Bearer undefined` (or something like this) header?

Reply:
Line 18 in nexus-sync.ts would prevent that in practice. We always either enter here with a token or a username and password.